build/commands/lib/config.js

```diff
@@ -182,6 +182,7 @@ const Config = function () {
   this.nativeRedirectCCDir = path.join(this.srcDir, 'out', 'redirect_cc')
   this.use_goma = getNPMConfig(['brave_use_goma']) || false
   this.goma_offline = false
+  this.build_sparkle = false
```

Comment:
Defaults should be set in gn, not config.js and `build_sparkle` can be set with `--gn`

Reply:
Resolved in cddafe61ee2bd06c379bd19a5346181274f02094.